network/base/models.py

```diff
@@ -259,7 +260,7 @@ class Data(models.Model):
     vetted_datetime = models.DateTimeField(null=True, blank=True)
     vetted_user = models.ForeignKey(User, related_name="vetted_user_set", null=True, blank=True)
     vetted_status = models.CharField(choices=OBSERVATION_STATUSES,
-                                     max_length=10, default='unknown')
+                                     max_length=20, default='unknown')
```

Comment:
The 'data_not_verified' choice would not fit in 10 chars and was getting truncated.